packages/mcp/src/context-assembly/assembler.ts

```
@@ -0,0 +1,578 @@
+    return Math.round(score * 100) / 100
+  }
+
+  /**
+   * Filter memories by relevance to participant
+   */
+  private async filterRelevantMemories(
+    memories: ExtractedMemory[],
+    participantId: string
+  ): Promise<ExtractedMemory[]> {
+    return memories.filter(memory => {
+      const isParticipant = memory.participants.some((p: any) => p.id === participantId)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)